cmd/mesh/manifest-migrate.go

```diff
@@ -16,22 +16,78 @@ package mesh
 
 import (
 	"fmt"
+	"io/ioutil"
+	"os"
 
+	"github.com/ghodss/yaml"
 	"github.com/spf13/cobra"
+
+	"istio.io/operator/pkg/apis/istio/v1alpha2"
+	"istio.io/operator/pkg/translate"
+	"istio.io/operator/pkg/version"
 )
 
-func manifestMigrateCmd(rootArgs *rootArgs) *cobra.Command {
+type manifestMigrateArgs struct {
+	// inFilename is the path to the input values.yaml file.
+	inFilename string
+	// outFilename is the path to the translated output filename.
+	outFilename string
```

Comment:
I think it would be better to require a user to specify a file and output to stdout because this is the only mode for this command. Having these as flags implies that they are optional.

Reply:
Can you help clarify two points here, 1. we have a pending issue if we do not specify a input file then we need to read the specs from cluster directly, so input seems like an optional field? 2. output file is also an option if user wants to write it to new file

Reply:
1. you could just as easily add the default to be cluster config if no file is specified. It does not need to be through -f flag. 
2. -o seems very limited value since it's just command > output file. Most linux commands follow this pattern of not having a -o flag if it can be supported with file redirect e.g. see helm template --help. manifest generate only has -o because it outputs to dirs.

Reply:
yep that makes sense